src/audio/pcm_converter/pcm_converter_hifi3.c

```diff
@@ -26,7 +26,7 @@
  * \brief Sets buffer to be circular using HiFi3 functions.
  * \param[in,out] buffer Circular buffer.
  */
-static void pcm_converter_setup_circular(struct comp_buffer *buffer)
+static void pcm_converter_setup_circular(const struct comp_buffer *buffer)
```

Comment:
It may also be beneficial here for users if we additionally rename buffer to source to further make it obvious it's RO data. Most of the other function updated here already have the name change afaict.

Reply:
This function is used to source and sink buffer, but it doesn't modify any buffer field itself, so const keyword is not misleading here.

Reply:
sorry, I mean then renaming would further help anone looking at this code (alongside th euse to conts keyword) to make it obvious it's RO data as far as this user is concerned.

Reply:
changed